### PR TITLE
Organic Interface Update Refactor Pt. 5

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Okay, every item's refactored, now onto clothing, fun.
This refactors everything above lewd_shoes.dm aside from the BDSM mask, deprivation helmet, dominant cap, kinky sleeping bag, catsuit, and the glasses.

TODO:

- [x] Test this shit

## How This Contributes To The Skyrat Roleplay Experience
So many things need to be redone.

## Changelog

:cl:
spellcheck: Fixed many spelling errors in organic interface messages
refactor: Refactored some organic interface clothing
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
zamn, this hits hard

**Comment by optimumtact** [comment]
N000

**Comment by Zonespace27** [comment]
> N000

Went ahead and finished this, since you seem to like it so much 😉

**Comment by optimumtact** [comment]
thank you

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/hypnogoggles.dm [hunk 1]
```diff
@@ -7,42 +7,50 @@
...
-			victim.gain_trauma(new /datum/brain_trauma/induced_hypnosis(codephrase), TRAUMA_RESILIENCE_BASIC)
-		else
-			codephrase = "Obey"
-			victim.gain_trauma(new /datum/brain_trauma/induced_hypnosis(codephrase), TRAUMA_RESILIENCE_BASIC)
+	if(!(iscarbon(victim) && victim.client?.prefs?.read_preference(/datum/preference/toggle/erp/sex_toy)))
+		return
+	if(codephrase != "")
+		victim.gain_trauma(new /datum/brain_trauma/induced_hypnosis(codephrase), TRAUMA_RESILIENCE_BASIC)
+	else
+		codephrase = "Obey"
```
Comment (Funce) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/hypnogoggles.dm]: This should include a period to be consistent with the default.

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/kink_collars.dm [hunk 2]
```diff
@@ -117,57 +120,49 @@
...
-		if(key.key_id==collar.key_id)
-			if(lock != FALSE)
-				IsLocked(FALSE,user)
-			else
-				IsLocked(TRUE,user)
-		else
-			to_chat(user, span_warning("This isn't the correct key!"))
-	return
+		return
+	to_chat(user, span_warning("It looks like the lock is brokenn - now it's just an ordinary old collar."))
```
Comment (Funce) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/kink_collars.dm]:
```suggestion
	to_chat(user, span_warning("It looks like the lock is broken - now it's just an ordinary old collar."))
```

**Review by Funce — COMMENTED** [review]
Apart from below, looks good. Once tested, we'll get this on the way.

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/hypnogoggles.dm (hunk 1 above)
Comment (Funce) [reply modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/hypnogoggles.dm]:
```suggestion
		codephrase = "Obey."
```

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**